tests/toolkit/matl_sell_policy_tests.cc

```diff
@@ -186,20 +186,95 @@ TEST_F(MatlSellPolicyTests, Package) {
   EXPECT_NO_THROW(sim.Run());
 
   QueryResult qr_trans = sim.db().Query("Transactions", NULL);
-  QueryResult qr_res = sim.db().Query("Resources", NULL);
   EXPECT_EQ(3, qr_trans.rows.size());
 
   EXPECT_EQ(0, qr_trans.GetVal<int>("Time", 0));
-  EXPECT_EQ(0, qr_trans.GetVal<int>("Time", 1));
-  EXPECT_EQ(1, qr_trans.GetVal<int>("Time", 2));
+  EXPECT_EQ(1, qr_trans.GetVal<int>("Time", 1));
+  EXPECT_EQ(2, qr_trans.GetVal<int>("Time", 2));
+
+  // Resource 0 is the material of 5 that we first created. 
+  QueryResult qr_all_res = sim.db().Query("Resources", NULL);
+  EXPECT_EQ(5, qr_all_res.GetVal<double>("Quantity", 0));
+
+  // Then check that three packaged materials were created of size 2, 2, and 1
+  std::vector<cyclus::Cond> conds;
+  conds.push_back(cyclus::Cond("PackageName", "==", std::string("foo")));
+  QueryResult qr_pkgd_res = sim.db().Query("Resources", &conds);
+  EXPECT_EQ(qr_pkgd_res.rows.size(), 3);
 
-  // Resource 0 is the material of 5 that we first created. Resource 1/2 is 
-  // the first resource split into 3 and 2. Resource 3/4 is split into 1 and 2
-  EXPECT_NEAR(2, qr_res.GetVal<double>("Quantity", 2), 0.00001);
-  EXPECT_NEAR(2, qr_res.GetVal<double>("Quantity", 4), 0.00001);
+  std::vector<double> pkged_res = {qr_pkgd_res.GetVal<double>("Quantity", 0),
+                                   qr_pkgd_res.GetVal<double>("Quantity", 1),
+                                   qr_pkgd_res.GetVal<double>("Quantity", 2)};
+  std::sort(pkged_res.begin(), pkged_res.end(), std::greater<double>());
+
+  std::vector<double> exp = {2, 2, 1};
+
+  EXPECT_EQ(exp, pkged_res);
 
   // All material should have been transacted, including the resource of size 1
   EXPECT_EQ(0, buf.quantity());
 }
+
+TEST_F(MatlSellPolicyTests, TransportUnit) {
+  using cyclus::QueryResult;
+
+  int dur = 5;
+  double throughput = 1;
+
+  cyclus::MockSim sim(dur);
+  cyclus::Agent* a = new TestFacility(sim.context());
+  
+  sim.context()->AddPrototype(a->prototype(), a);
+  sim.agent = sim.context()->CreateAgent<cyclus::Agent>(a->prototype());
+  sim.AddSink("commod").Finalize();
+  TestFacility* fac = dynamic_cast<TestFacility*>(sim.agent);
+
+  cyclus::toolkit::ResBuf<cyclus::Material> buf;
+
+  double qty = 5;
+  CompMap cm;
+  cm[922350000] = 0.05;
+  cm[922380000] = 0.95;
+  Composition::Ptr comp = Composition::CreateFromMass(cm);
+  mat = Material::Create(a, qty, comp, Package::unpackaged_name());
+
+  buf.Push(mat);
+
+  sim.context()->AddPackage("foo", 1, 2, "first");
+  Package::Ptr p = sim.context()->GetPackage("foo");
+  sim.context()->AddTransportUnit("foo", 3, 3, "first");
```

Comment:
might be helpful to call the TU a different name than the Pkg test readability, even though it doesn't matter if their names collide